src/pages/RFOX/components/Claim/ClaimRow.tsx

```diff
@@ -0,0 +1,120 @@
+import { Box, Button, Flex, Tooltip } from '@chakra-ui/react'
+import { foxAssetId, foxOnArbitrumOneAssetId } from '@shapeshiftoss/caip'
+import { bnOrZero } from '@shapeshiftoss/chain-adapters'
+import { TransferType } from '@shapeshiftoss/unchained-client'
+import { type FC, useCallback, useMemo } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { useHistory } from 'react-router'
+import { Amount } from 'components/Amount/Amount'
+import { AssetIconWithBadge } from 'components/AssetIconWithBadge'
+import { RawText } from 'components/Text'
+import { TransactionTypeIcon } from 'components/TransactionHistory/TransactionTypeIcon'
+import { toBaseUnit } from 'lib/math'
+import { selectAssetById, selectFirstAccountIdByChainId } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import { ClaimRoutePaths, ClaimStatus, type RfoxClaimQuote } from './types'
+
+type ClaimRowProps = {
+  amountCryptoPrecision: string
+  status: ClaimStatus
+  setConfirmedQuote: (quote: RfoxClaimQuote) => void
+  cooldownPeriodHuman: string
+  index: number
+}
+
+const hoverProps = { bg: 'gray.700' }
+
+export const ClaimRow: FC<ClaimRowProps> = ({
+  amountCryptoPrecision,
+  status,
+  setConfirmedQuote,
+  cooldownPeriodHuman,
+  index,
+}) => {
+  const translate = useTranslate()
+  const history = useHistory()
+
+  const stakingAsset = useAppSelector(state => selectAssetById(state, foxOnArbitrumOneAssetId))
+  const stakingAssetSymbol = stakingAsset?.symbol
+  const stakingAmountCryptoBaseUnit = toBaseUnit(
+    bnOrZero(amountCryptoPrecision),
+    stakingAsset?.precision ?? 0,
+  )
+
+  // TODO(apotheosis): make this programmatic when we implement multi-account
+  const stakingAssetAccountId = useAppSelector(state =>
+    selectFirstAccountIdByChainId(state, stakingAsset?.chainId ?? ''),
+  )
+
+  const claimQuote: RfoxClaimQuote = useMemo(
```

Comment:
Do we want to also use `stakingAsset` terminology for those fields?

Reply:
Also this is only used in `handleClaimClick` - we could declare it there instead and avoid the opt-chain on `stakingAssetAccountId` by early returning

Reply:
Addressed in https://github.com/shapeshift/web/pull/7073/commits/5376b791a13ffb5ddd68e707a52f83cd6bdb864e 👌